packages/drafts-realm/contact-form.gts

```diff
@@ -0,0 +1,250 @@
+import { UserName } from './user-name';
+import { UserEmail } from './user-email';
+import { CardDef, field, contains } from 'https://cardstack.com/base/card-api';
+import { Component } from 'https://cardstack.com/base/card-api';
+import StringField from 'https://cardstack.com/base/string';
+
+import {
+  FieldContainer,
+  CardContainer,
+  IconButton,
+} from '@cardstack/boxel-ui/components';
+import { Sparkle } from '@cardstack/boxel-ui/icons';
+import { tracked } from '@glimmer/tracking';
+
+class Isolated extends Component<typeof ContactForm> {
+  @tracked hasTitleField =
```

Comment:
Use a getter. tracked would be a good choice if you set something to it, but since it's a derived (computed) value it should be a `get hasTitleField() {}`

Reply:
You don't have to show screenshots of code changes. They are clearly visible in the commits. With screenshots I meant showing the changes in application's interface (i.e. demonstrate the features of your card using screenshots or video)